### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by ftramer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ If you use this paper in academic research, you may cite the following:
 
 ```
 @article{carlini2019evaluating,
-  title={},
+  title={On Evaluating Adversarial Robustness},
```
Comment (carlini) [inline README.md:50]: The current README already has this change I believe.

Comment (ftramer) [reply README.md:50]: Yep indeed. Seems like my page hadn't refreshed properly

Comment (carlini) [reply README.md:50]: It's strange that the merge would work, right? Shouldn't this conflict?

Comment (anishathalye) [reply README.md:50]: Trying it out locally, it seems to work fine:

```
$ git merge ftramer/patch-1
Auto-merging README.md
Merge made by the 'recursive' strategy.
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
```

Comment (ftramer) [reply README.md:50]: Given that it's the exact same change there's no actual conflict, right? Git usually merges these fine (as above in Anish's attempt).

**Pull request merged**